### PR TITLE
🌐 Add Portuguese translation of `tutorial/extra-data-types.md`

### PR DESCRIPTION
**Pull request by luccasmmg**

### CODE REVIEW

File: docs/pt/docs/tutorial/extra-data-types.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Tipos de dados extras
+
+Até agora, você tem usado tipos de dados comuns, tais quais:
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/extra-data-types.md:3]:
```suggestion
Até agora, você tem usado tipos de dados comuns, tais como:
```

File: docs/pt/docs/tutorial/extra-data-types.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Mas você também pode usar tipos de dados mais complexos.
+
+E você ainda terá os mesmos recursos que viu até agora:
+
+* Ótimo suporte do editor.
+* Conversão de dados das requisições recebidas.
+* Conversão de dados para os dados da resposta.
+* Validação de dados.
+* Anotação e documentação automática.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/extra-data-types.md:18]:
```suggestion
* Anotação e documentação automáticas.
```

File: docs/pt/docs/tutorial/extra-data-types.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    * Em requisições e respostas será representado como uma `str` no formato ISO 8601, exemplo: `2008-09-15`.
+* `datetime.time`:
+    * O `datetime.time` do Python.
+    * Em requisições e respostas será representado como uma `str` no formato ISO 8601, exemplo: `14:23:55.003`.
+* `datetime.timedelta`:
+    * O `datetime.timedelta` do Python.
+    * Em requisições e respostas será representado como um `float` de segundos totais.
+    * O Pydantic também permite representá-lo como uma "codificação ISO 8601 diferença de tempo", <a href="https://pydantic-docs.helpmanual.io/#json-serialisation" class="external-link" target="_blank">cheque a documentação para mais informações</a>.
+* `frozenset`:
+    * Em requisições e respostas, vai ser tratado da mesma forma que um `set`:
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/extra-data-types.md:41]:
```suggestion
    * Em requisições e respostas, será tratado da mesma forma que um `set`:
```

File: docs/pt/docs/tutorial/extra-data-types.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        * Nas requisições, uma lista vai ser lida, eliminando duplicadas e convertendo-se a um `set`.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/extra-data-types.md:42]:
```suggestion
        * Nas requisições, uma lista será lida, eliminando duplicadas e convertendo-a em um `set`.
```

File: docs/pt/docs/tutorial/extra-data-types.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    * Em requisições e respostas será representado como uma `str`.
+    * O esquema gerado vai especificar que é uma `str` com o "formato" `binary`.
+* `Decimal`:
+    * O `Decimal` padrão do Python.
+    * Em requisições e respostas será representado como um `float`.
+* Você pode checar todos os tipos de dados válidos do Pydantic aqui: <a href="https://pydantic-docs.helpmanual.io/usage/types" class="external-link" target="_blank">Tipos de dados do Pydantic</a>.
+
+## Exemplo
+
+Aqui está uma *operação de rota* exemplo com parãmetros utilizando-se de alguns dos tipos acima.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/extra-data-types.md:56]:
```suggestion
Aqui está um exemplo de *operação de rota* com parâmetros utilizando-se de alguns dos tipos acima.
```

File: docs/pt/docs/tutorial/extra-data-types.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+```Python hl_lines="1  3  12-16"
+{!../../../docs_src/extra_data_types/tutorial001.py!}
+```
+
+Note que os parâmetros dentro da função tem seu tipo de dados natural, e você pode, por exemplo, performar manipulações de data normais, como:
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/extra-data-types.md:62]:
```suggestion
Note que os parâmetros dentro da função tem seu tipo de dados natural, e você pode, por exemplo, realizar manipulações normais de data, como:
```

**Review by LorhanSohaky — COMMENTED** [review]
I think with these suggestions the text is more natural

**Review by LorhanSohaky — APPROVED** [review]

**Review by peidrao — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome, thank you @luccasmmg! :cake: 

And thanks for the reviews @LorhanSohaky and @peidrao ! :cookie:

**Pull request merged**